.github/workflows/integration-test-v2.yml

```diff
@@ -72,5 +77,14 @@ jobs:
              echo "engine_name=${{ steps.setup.outputs.engine_name }}" >> $GITHUB_OUTPUT
           fi
 
+      - name: Setup env by build
+        id: setup-env
+        run: |
+          if [ $runsetup == 'true' ]; then
```

Comment:
Typo? `$runsetup` is not defined anywhere.

Reply:
ups, 10x. I do not understand how I thought that it works correctly. Fixed